week-2/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,8 @@
 */
 
 var numbers = [2, 2, 3]; // Don't change this array literal declaration
-
+var numbers = numbers.concat(4);
```

Comment:
This works, but be careful about declaring `var numbers` twice as this sort of thing can be confusing if people are reading a lot of code, see `var something` and think that they're seeing the first declaration of `something`.

It does do the same thing though!